### PR TITLE
@craigspaeth => First pass at the /auction2 page using es6

### PR DESCRIPTION
**Pull request by sweir27**
(thanks @craigspaeth for the initial pair!) This is the first step in making a new, metaphysics-backed `/auction` page, written in es6 instead of coffeescript. 

For now, it just shows the above-the-fold content for a sale, and is `adminOnly`:
image

For now, the view mostly copies existing templates from the `auction` app. I added a basic view test, but will continue to add those as there is more logic.

### CODE REVIEW

**Comment by craigspaeth** [comment]
Looks like the shrinkwrap diff is off, try running `rm npm-shrinkwrap.json && npm shrinkwrap --dev` to see if it includes dev dependencies.

File: apps/auction2/routes.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import metaphysics from '../../lib/metaphysics'
+import Auction from '../../models/auction.coffee'
+
+export const index = async (req, res) => {
+  const user = req.user
+  const {me} = await metaphysics({
```
Comment (craigspaeth) [inline apps/auction2/routes.js]: ❤️  async/await.

**Comment by kanaabe** [comment]
so fancy

**Comment by craigspaeth** [comment]
👏  Amazing! Great work! This is a very exciting rewrite and something we can use as a reference for pivoting how we build new sub-apps in Force. Good job adding the babel configs throughout the project! Looking forward to seeing more tests and comparing the code diff between the two apps.

File: package.json [hunk 2]
```diff
@@ -3,13 +3,22 @@
   "version": "1.0.0",
   "private": true,
   "engines": {
-    "node": ">=5.11.1",
+    "node": "6.x.x",
```
Comment (craigspaeth) [inline package.json]: ❤️

**Comment by acjay** [comment]
Awesome work @sweir27! Stack upgrade, here we come!

**Comment by sweir27** [comment]
@craigspaeth oops on the shrinkwrap! Added it back

File: apps/auction2/test/templates.js [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+      it('renders correctly', () => {
+        $('.auction2-title').text().should.equal('An Auction')
+        $('.auction2-header-metadata').text().should.containEql('Registration pending')
+        $('.auction2-header-metadata').text().should.containEql('Reviewing submitted information')
+      })
+    })
+
+    describe('index, registered to bid and qualified', () => {
+      before((done) => {
+        benv.setup(() => {
```
Comment (craigspaeth) [inline apps/auction2/test/templates.js]: Realizing benv should integrate a promise API now.

**Pull request merged**